wiki/Mascots/de.md

```diff
@@ -1,290 +1,101 @@
+![Alisa](img/Alisa.png "Alisa")
 
-#### Geschlecht
+Alisa wurde von [\[ Glitch \]](https://osu.ppy.sh/users/3781400) entworfen und war eine der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Sie spielt osu!taiko schon seit ihrer frühen Kindheit. Wenn sie nicht gerade schläft oder isst, spielt sie osu!taiko oder hat Spaß an Retro-Videospielen!
 
-männlich
+### Chirou
 
-#### Beigetreten
+![Chirou](img/Chirou.png "Chirou")
 
-May 2008
+Chirou wurde von [pyun](https://osu.ppy.sh/users/981534) entworfen und war eine der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Sie ist sehr streng und anspruchsvoll, eine Perfektionistin und macht nicht gerne Fehler—vor allem nicht bei ihren Beats in osu!taiko. Aber hinter ihrer strengen Fassade kann sie, wenn man an ihren weichen Punkt kommt, auch flauschig und süß sein. Sie ist 14 Jahre alt und wurde am 25. Oktober mit der Blutgruppe AB geboren. Außerdem ist sie 1,80 m groß und wiegt 45 Kilogramm. Zu ihren Hobbys gehören das Trommeln, sich in ihrem Umhang warmzuhalten und Edelsteine oder Steine zu sammeln. Chirou hat einige Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
 
-#### Gewicht
+### Taikonator
 
-100+ kg
+![Taikonator](img/Taikonator.png "Taikonator")
 
-#### Biologische Daten
+Taikonator, auch bekannt als Taikonator 3000, war einer der Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko. Er hat aus unbekannten Gründen als Insider-Witz eine gewisse Zugkraft und Popularität erlangt. Seine Herkunft bleibt ein Rätsel; trotzdem ist er weitaus einzigartiger als die anderen Designvorschläge für das osu!taiko-Maskottchen und hat viele Auftritte in verschiedenen Fanart-Beiträgen von Benutzern.
 
-Don ist der Protagonist der Taiko no Tatsujin Serie. Er ist die Taiko-Trommel mit weißem Rand und vier Beinen, einem roten Gesicht (welches teils in Richtung Orange geht) und einem hell blauen Körper. Dons Traum ist es, die Schönheit von Taiko mit der ganzen Welt zu teilen. 3 Jahre sind nun schon vergangen, nachdem er ins Wada-Haus eingezogen ist und mehr und mehr in der Stadt bekannt wurde. Er hat einen monströsen Appetit und gibt viel Geld beim Einkaufen im Wada-Haus aus, welches nicht zu seinen Guten Seiten zählt. Er beendet für gewöhnlich seine Sätze mit "Ta-don", was aus dem Japanischen für "Ba-dum" steht.
+### Tama
 
-## Bildergalerie
+![Tama](img/Tama.png "Tama")
 
-[Klicke hier drauf, um zur Bildergalerie zu gelangen.](/wiki/Mascots/Gallery)
+Tama wurde von [crystalsuicine](https://osu.ppy.sh/users/9974) entworfen und war mit nur 15 Jahren eine der jüngeren Teilnehmenden am Designwettbewerb für das Maskottchen von osu!taiko (oder doch nicht?). Tama hat eine große Leidenschaft für das Taiko-Trommeln. Und Gewitter. Und vor allem Festivals, auf denen sie jedes Takoyaki aufschnappt, das sie finden kann. Tama ist immer für eine Herausforderung zu haben und verbirgt hinter ihrem jugendlichen Auftreten eine besonders mysteriöse Vergangenheit.
```

Comment:
It might be that the `or is she?` wants to ask if she is actually 15. However, the German text conveys more of a `Is she the youngest participant?`. I can't think of a better version here, so I think it is still okay. It's just something that I've noticed.

Reply:
Yeah this one is kinda weird, so i just took the literal translation